Rex.podspec

```diff
@@ -18,8 +18,8 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target = '9.0'
 
-  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :tag => s.version }
-  s.dependency 'ReactiveCocoa', '~> 4.1'
+  s.source       = { :git => 'https://github.com/neilpa/Rex.git', :commit => '265e8e9c9503bc917ee5af5252590ab38eda4bff' }
```

Comment:
No need to change this line. We should simply have @neilpa cut a new version. It is a little impractical we are so heavily tied to to the RAC version.

It fails linting due to:

```
-> Rex (0.11.0)
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACCompoundDisposable.m:85:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACCompoundDisposable.m:132:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACSerialDisposable.m:63:12: warning: unused variable 'result' [-Wunused-variable]
    - WARN  | xcodebuild:  ReactiveCocoa/ReactiveCocoa/Objective-C/RACSerialDisposable.m:79:12: warning: unused variable 'result' [-Wunused-variable]